### PR TITLE
WebAPI: Update Method pages to modern structure (part 20) 

### PR DESCRIPTION
**Pull request by OnkarRuikar**
Continuing #14857 

## Summary
Updating the method pages to follow the current templates:
https://developer.mozilla.org/en-US/docs/MDN/Structures/Syntax_sections#constructors_and_methods
https://developer.mozilla.org/en-US/docs/MDN/Structures/Page_types/API_method_subpage_template

Changes Include:
- fix syntax section
- changing section titles to suggested titles in the template
- change `{{DOMxRef("DOMString")}}` to `string`
- remove extra spaces in between words and extra trailing spaces
- other small corrections

### Metadata
- [x] Fixes a typo, bug, or other error

### CODE REVIEW

File: files/en-us/web/api/writablestreamdefaultwriter/abort/index.md [hunk 1]
```diff
@@ -32,7 +32,7 @@ abort(reason)
 ### Parameters
 
 - reason {{optional_inline}}
-  - : A string representing a human-readable reason for the abort.
+  - : A {{domxref("DOMString")}} representing a human-readable reason for the abort.
```
Comment (teoli2003) [inline files/en-us/web/api/writablestreamdefaultwriter/abort/index.md:35]: Why this change?

**Review by teoli2003 — CHANGES_REQUESTED** [review]
One question to answer and this is good to go.

File: files/en-us/web/api/writablestreamdefaultwriter/abort/index.md (hunk 1 above)
Comment (OnkarRuikar) [reply files/en-us/web/api/writablestreamdefaultwriter/abort/index.md:35]: It got reverted in merge.  :(
```suggestion
  - : A string representing a human-readable reason for the abort.
```

**Comment by OnkarRuikar** [comment]
@teoli2003 done!
It got reverted during conflict resolution.

**Review by teoli2003 — APPROVED** [review]
👍
